.github/workflows/prepare-release.yaml

```diff
@@ -111,7 +111,14 @@ jobs:
              - Bug fixes
              - Any changes in behaviour
              - Changes requiring upgrade notices or deprecation warning
-
+            
+            ## Flux compatibility  
+             | Flux version | Minimum required |
+             |--------------------|------------------|
+             | `v2.0`            | `>= 2.0.0`   |
+            
+             For Flux migrations to v2.0 see [flux](https://github.com/fluxcd/flux2/releases/tag/v2.0.0) or [weave gitops](https://docs.gitops.weave.works/docs/next/guides/fluxga-upgrade/) documentation.
```

Comment:
The link should be https://docs.gitops.weave.works/docs/guides/fluxga-upgrade/

Reply:
😅  updated!